client/App.jsx

```diff
@@ -1,17 +1,18 @@
-import React, { Component } from 'react';
-import { Redirect, Route, Switch } from 'react-router-dom';
```

Comment:
I think we want to keep importing redirect, right?

Reply:
this was a duplication

Reply:
you are right nm